js/algoliasearch/internals/frontend/common.js

```diff
@@ -336,12 +336,21 @@ document.addEventListener("DOMContentLoaded", function (e) {
 
 					var ors = '';
 
-					if (keys.length > 0) {
-						ors += '<span><a href="' + keys[0].url + '">' + keys[0].key + '</a></span>';
+					var isCategoryFacetEnabled = false;
```

Comment:
Right now, the conditions happening no matter if there are some `keys` or not, which is not super effective. Can it be run only if some keys are present? Or can `keys` be retrieved only if categories are in facets? It's not effective to do both :)